safe_transaction_service/history/services/index_service.py

```diff
@@ -104,10 +104,15 @@ def is_service_synced(self) -> bool:
             erc20_block_number__lt=reference_block_number
         ).count()
         if out_of_sync_contracts > 0:
-            logger.error(
-                "%d Safe Contracts have ERC20/721 out of sync", out_of_sync_contracts
-            )
-            synced = False
+            total_number_of_contracts = SafeContract.objects.all().count()
+            proportion_out_of_sync = out_of_sync_contracts / total_number_of_contracts
+            # Ignore less than 10% of contracts out of sync
+            if proportion_out_of_sync >= 0.1:
```

Comment:
I think we can have this as an environment variable what do you think?

Reply:
Makes sense